src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20201120140950_Routine_RemoveSoftDeletedSubjects.sql

```diff
@@ -68,13 +68,20 @@ BEGIN
                                      WHERE Filter.Id = OFI.FilterId
                                        AND Filter.SubjectId IN (SELECT Id FROM #SoftDeletedSubjects));
 
+                        DELETE TOP (@ObservationFilterItemCommitBatchSize) OFI
```

Comment:
You are missing a migration for this change.

Reply:
Hey @benoutram, not sure what you think might be missing?  The ObservationRowFilterItem table is getting created in the "20210125164907_EES1830_AddNewObservationTables.cs" migration, unless you were referring to something else?

Reply:
I can't see what is reloading `20201120140950_Routine_RemoveSoftDeletedSubjects.sql` i.e. updadting  `RemoveSoftDeletedSubjects`, and as you know at the moment the convention is to duplicate the source and alter it, although I agree with your suggestion in future we could have a _current_ and _previous_ for all of them.